### ISSUE TITLE
Change root.returnExports to root.styleSelect in UMD

### ISSUE THREAD

**Issue by adrianfalleiro** [mikemaccana/styleselect#8]
In the UMD block below:

``` js
(function (root, factory) {
    if (typeof define === 'function' && define.amd) {
        // AMD. Register as an anonymous module.
        define([], factory);
    } else if (typeof exports === 'object') {
        // Node. Does not work with strict CommonJS, but
        // only CommonJS-like environments that support module.exports,
        // like Node.
        module.exports = factory();
    } else {
        // Browser globals (root is window)
        root.returnExports = factory();
  }
}(this, function () {
```

When the factory is attached to the root (window) it is attached to the property `root.returnExports` - this means that it is access via `window.returnExports`

Shouldn't this be `root.styleSelect`?

**Comment by bdavidxyz** [comment]
Indeed, it should. The lib doesn't work for the browser without this modification.

**Comment by neuropass** [comment]
argghh!!! @mikemaccana merge this? https://github.com/mikemaccana/styleselect/pull/9

**Comment by mikemaccana** [comment]
Fixed in 1.0.3 and published
